### PR TITLE
Changing streamable episode ordering

### PR DESCRIPTION
**Pull request by votemike**
To fix this problem: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5737?focusedCommentId=3613303&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3613303

### CODE REVIEW

File: src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php [hunk 1]
```diff
@@ -232,28 +232,32 @@ public function findDescendantsByType(array $ancestryDbIds, string $entityType,
...
+     * @param int[] $ancestryDbIds
+     * @param string $entityType
+     * @param DateTimeImmutable $now
      * @param int|null $limit
-     * @param int      $offset
+     * @param int $offset
      * @return array
      */
-    public function findStreamableDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
+    public function findStreamableDescendantsByType(array $ancestryDbIds, string $entityType, DateTimeImmutable $now, ?int $limit, int $offset) : array
```
Comment (felipeparaujo) [inline src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php]: I would call this cutoffTime to be in line with the rest of the stuff

Comment (votemike) [reply src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php]: But it's not the cutoff time. I could do the 7 day logic in the Service I suppose. Then pass it through?

**Comment by votemike** [comment]
@BPScott not keen on those sort of tests. Would that not be better as an automated test by the testers?

**Comment by votemike** [comment]
Test this please

**Pull request merged**